Issue #176 … (#182)
cncf/sig-security: assessments/guide/outline.md

## Project Design
* Design.  A description of the system design that discusses how it works.
This is especially critical for any mechanisms that relate to the security
of the system.  Include architecture and network (if applicable) information such as encryption of traffic between services, access control types (RBAC, etc.) and enforcement, or security logging, etc.

## Configuration and Set-Up
* Default.  Documentation describing the default configuration of the project with initial set-up instructions (link to docs is acceptable). Documentation should identify potential security risks/trade-offs of the default config.
* Secure. Documentation describing recommended secure configuration and set-up instructions, beyond defaults with justification for selection and trade-offs (link to docs is acceptable).
* Advanced Secure. If applicable, documentation describing advanced settings for most hardened configuration of the project to include justification for selection and trade-offs (link to docs is acceptable).

## Project Compliance
* Compliance.  List any security standards or sub-sections the project is already documented as meeting (PCI-DSS, COBIT, ISO, GDPR, etc.).

> 

Reviewer: Really like the additions of config/setup and compliance.   I believe this will help projects think to write up relevant aspects of the project that they might otherwise forget to include.

## Security Analysis

> * Attacker Motivations.  A discussion about the likely goals of an attacker as well as the kind of attacker (do not forget to invlude discussion of insider threat with trusted access to the project)

Reviewer 2: ```suggestion
* Attacker Motivations.  A discussion about the likely goals of an attacker as well as the kind of attacker (do not forget to include discussion of insider threat with trusted access to the project)
```
Minor correction invlude->include

This likely relates closely to the impact of different attacks in the
scenarios.  (e.g., In the password hash case, the attacker wants to expose
those hashes on the Flibble server.  However, a Flibble cloudlet attacker
may find it more interesting to bring down the service.)

> * Predisposing Conditions. A listing of potential vulnerabilities and configurations of the project that could potentially be exploited or used correctly to result in an increased likelihood of attack success. Include any established trusts with other projects as a potential vulnerable point from a pivoted attack (i.e. compromise of the LDAP results in loss of access control integrity for the project)

Reviewer 3: Slightly awkward phrasing here.
```suggestion
* Predisposing Conditions. A list of potential vulnerabilities and configurations of the project that could potentially be exploited or used correctly to result in an increased likelihood of attack success. Include any trust relationships with other projects that pose a risk of compromise for this project (i.e. compromise of the LDAP results in loss of access control integrity for the project)
```

> 

Author: ## Security Issue Resolution
* Responsible Disclosures Process.  A outline of the project's responsible disclosures process should suspected security issues, incidents, or vulnerabilities be discovered both external and internal to the project.  The outline should discuss communication methods/strategies.
* Incident Response.  A description of the defined procedures for triage, confirmation, notification of vulnerability or security incident, and patching/update availability.
